app/assets/stylesheets/styles/guides/_new_guides.css.scss

```diff
@@ -1,5 +1,4 @@
-// You can't nest keyframes
-
+// TODO: This should not be here. This makes me unhappy.
 @-webkit-keyframes fa-spin-image-load {
```

Comment:
Yea it was weird I tried moving this to `main` but it didn't seem to be detected in `_new_guides`…